app/src/main/java/com/meewii/rateconverter/business/database/ExchangeRateDao.kt

```diff
@@ -30,7 +30,7 @@ interface ExchangeRateDao {
 
   @Transaction
   @Query("SELECT * FROM $EXCHANGE_RATE_TABLE_NAME WHERE $EXCHANGE_RATE_COLUMN_BASE_CURRENCY_ID = :baseCurrencyId")
-  fun getRatesForBase(baseCurrencyId: String): Maybe<ExchangeRateEntity>
```

Comment:
I truly hate maybes :D :D

Reply:
well it's gone :D I hope it made you happy :)